react.js

```diff
@@ -9,13 +9,14 @@ module.exports = {
         'plugin:prettier/recommended', // Enables eslint-plugin-prettier and displays prettier errors as ESLint errors. Should be last in the extends array.
     ],
     plugins: ['react-hooks'],
+    rules: {
+        'react/jsx-sort-props': 'error',
```

Comment:
If we add `sort-key`, we should probably turn on this as well.

Reply:
Auto-fixed by Prettier?